module/components/Grid/index.js

```diff
@@ -98,19 +179,28 @@ export class Grid extends React.Component {
                     md={md}
                     lg={lg}
                     xl={xl}
-                    className={classNames(className, this.renderJustifyContent(justify))}
+                    className={classNames(className)}
                     {...attributes}
                 >
                     { this.props.children }
```

Comment:
Remove `this.props` from here and put `children` prop to above props const object